src/index.html

```diff
@@ -25,7 +25,7 @@ <h4>Today's To-do</h4>
     <button id="add">Add</button>
    </div>
     <hr class="hr2">
-    <ul class="draggable-list" id="toDoList">
+    <ul class="container" id="toDoList">
     </ul>
     <button id="clear">Clear all Completed</button>
```

Comment:
- [ ] Kindly follow the requirements and Implement this feature because one of the requirements says that `Implement a function for the "Clear all completed" button (use filter() method).

- [ ] Also make sure you solve this issue, the remove button is not deleting the book in the localstorage. Kindly fix it

https://user-images.githubusercontent.com/65068771/172985532-fc3d3865-ab0c-4da4-8781-af9875ef6c41.mp4

Reply:
https://user-images.githubusercontent.com/71286979/172991901-d4da261e-1ad2-42b6-9be0-8c37dcbc11b7.mp4